templates/zerver/api/construct-narrow.md

```diff
@@ -0,0 +1,86 @@
+# Construct a narrow
+
+A **narrow** is a set of filters for Zulip messages, that can be based on many
+different factors (like sender, date, stream...).
+
+Narrows look like this:
+
+```
+stream:announce sender:iago@zulip.com
+```
+
+## Operators and operands
+
+Each of the key-value pairs that make a narrow are a filter for a specific
+characteristic of the messages. Here are the message filters (or
+**operators**) you can use for a narrow:
+
+* `search`: contain specific keywords. Example: `search:cake`.
+* `stream`: sent to a specific stream. Example: `stream:sales`.
+* `topic`: sent to a specific topic. Example: `topic:party`.
+* `sender`: sent by a specific user.
+    Example: `sender:aaron@zulip.com`.
+* `is`: belong to one of the following categories.
+    Example: `is:private`.
+    * `private`: sent to this user as a PM.
+    * `starred`: starred by the user making the narrow.
+    * `unread`: have not been read by the user making the narrow.
+    * `alerted`: contain one or more of the alert words configured by
+        the user making the narrow.
+    * `mentioned`: contain an `@mention` for the user making the narrow.
+* `has`: contain one or more of the following. Example: `has:link`.
+    * `link`
+    * `image`
+    * `attachment`
+* `near`: sent around some other. The other message has to be specified with
+    its ID. Example: `near:314`.
```

Comment:
Do we need to replicate the content from https://zulipchat.com/help/search-for-messages?  I think we could just end up linking there for this block.